src/Analysis/GWASUIMockup/GWASUI.jsx

```diff
@@ -0,0 +1,483 @@
+                    },
+                  ]}
+                >
+                  <InputNumber min={1} max={10} />
+                </Form.Item>
+                <Form.Item
+                  label='Covariates'
+                  name='covariates'
+                >
+                  <Select
+                    mode='multiple'
+                    disabled
+                    style={{ width: '100%' }}
+                  />
+                </Form.Item>
+                <Form.Item
+                  label='Outcome'
+                  name='outcome'
+                >
+                  <Input
+                    disabled
+                    style={{ width: '100%' }}
+                  />
+                </Form.Item>
+                <Form.Item
+                  label='Is Binary Outcome?'
+                  name='isBinary'
+                >
+                  <Input
+                    disabled
+                    style={{ width: '100%' }}
+                  />
+                </Form.Item>
+              </Form>
+            </div>
+          </Space>
+        );
+      }
+      case 4: {
+        const layout = {
+          labelCol: { span: 8 },
+          wrapperCol: { span: 16 },
+        };
+        const tailLayout = {
+          wrapperCol: { offset: 8, span: 16 },
+        };
+
+        if (showJobSubmissionResult) {
+          return (
+            <div className='GWASUI-mainArea'>
+              <Result
+                status={(jobSubmittedRunID) ? 'success' : 'error'}
+                title={(jobSubmittedRunID) ? 'GWAS Job Submitted Successfully' : 'GWAS Job Submission Failed'}
+                subTitle={`GWAS Job Name: ${jobName}, run ID: ${jobSubmittedRunID}`}
+                extra={[
+                  <Button
+                    key='done'
+                    onClick={() => {
+                      setCurrent(0);
+                      setSelectedCohort(undefined);
+                      setSelectedConcepts([]);
+                      setSelectedPhenotype(undefined);
+                      setSelectedCovariates([]);
+                      setNumOfPC(3);
+                      setJobName(undefined);
+                      setShowJobSubmissionResult(false);
+                      setJobSubmittedRunID(undefined);
+                    }}
+                  >
+                    Done
+                  </Button>,
+                ]}
+              />
+            </div>
+          );
+        }
+
+        return (
+          <div className='GWASUI-mainArea'>
+            <Form
+              {...layout}
+              name='control-hooks'
+              onFinish={(values) => {
+                setJobName(values.GWASJobName);
+                setJobSubmittedRunID('run-12345');
+                setMarinerJobStatus([
+                  ...marinerJobStatus,
+                  {
+                    runID: 'run-12345',
+                    status: 'running',
+                  },
+                ]);
```

Comment:
this is still just creating a dummy mariner job, not talking to mariner and launch a real GWAS job, is this intended?

Reply:
intended for now in GWASUIApp